doc/_templates/navbar.html

```diff
@@ -9,6 +9,7 @@
   <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
     <li><a href="https://mne.tools/mne-bids/dev/index.html">Development</a></li>
     <li><a href="https://mne.tools/mne-bids/stable/index.html">Stable</a></li>
+    <li><a href="https://mne.tools/mne-bids/v0.5/index.html">v0.5</a></li>
```

Comment:
just a note: this will lead to a 404 until the actual release is done, docs are built, and manually pushed to the gh-pages branch (see wiki "how to make a release")